modules/custom/activity_creator/js/notifications.js

```diff
@@ -15,27 +15,18 @@
     },
 
     _updateNotificationCount: function () {
-      var $notificationCount = $('.notification-bell .badge');
 
       // We won't proceed if the notification count is 0 or the dropdown is
       // already open.
       // @todo: Refactor to use data-* attributes and `jQuery.data()` to store
```

Comment:
We can remove this TODO because it's no longer applicable and should probably update the comment on the live above :)

Reply:
Updated.